public/editor-client/src/types/DefaultTemplate.ts

```diff
@@ -5,6 +5,15 @@ export interface DefaultBlock {
   value: Record<string, unknown>;
 }
 
+export interface AiGlobalStyles<T1, T2> {
```

Comment:
rename `AiGlobalStyles`

as i see in previous @maxval1 review there is nothing with AI